orttraining/orttraining/python/training/ortmodule/_logger.py

```diff
@@ -76,3 +77,70 @@ class LogColor:
     ENDC = "\033[0m"
     BOLD = "\033[1m"
     UNDERLINE = "\033[4m"
+
+
+class TimeTrackerPhase(IntEnum):
+    EndToEnd = 0
+    EXPORT = 1
+    GRAPH_BUILDER_INIT = 2
+    DETECTION = 3
+    BUILD_GRAPH = 4
+    CREATE_SESSION = 5
```

Comment:
Can we add comments to explain what each one means?

Reply:
Sure added.